src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -437,6 +437,21 @@ func (cfg *Config) applyClientDefaults() {
 		cfg.Client.ChainedServers = make(map[string]*client.ChainedServerInfo)
 	}
 
+	if cfg.Client.ProxiedCONNECTPorts == nil {
+		cfg.Client.ProxiedCONNECTPorts = []int{
+			// Standard HTTP(S) ports
+			80, 443,
+			// Common unprivileged HTTP(S) ports
+			8080, 8443,
+			// XMPP
+			5222, 5223, 5224,
+			// Android
+			5228, 5229,
+			// Google Hangouts (see https://support.google.com/a/answer/1279090?hl=en)
+			19302, 19303, 19304, 19305, 19306, 19307, 19308, 19309,
```

Comment:
IIRC Lantern mobile wraps all UDP traffic and forwards to udpgw waited at `127.0.0.1:7300` on server, so we should also add that port right? That will leave UDP traffic unlimited though, but we can add more restriction later.